spec/lib/ecco/row_event_listener_spec.rb

```diff
@@ -1,3 +1,6 @@
+java_import java.util.ArrayList
+java_import java.util.AbstractMap
+
```

Comment:
These can be moved to `event_context.rb` (Did not do that in https://github.com/twingly/ecco/pull/57#discussion_r524397737 because lazy :))

Reply:
👍  00b9350